docs/ref/forms/fields.txt

```diff
@@ -275,6 +275,35 @@ fields. We've specified ``auto_id=False`` to simplify the output:
     <div>Sender:<div class="helptext">A valid email address, please.</div><input type="email" name="sender" required></div>
     <div>Cc myself:<input type="checkbox" name="cc_myself"></div>
 
+When a field has help text and :attr:`~django.forms.BoundField.id_for_label`
+returns a value, we associate``help_text`` with the input using
+``aria-describedby``:
+
+    >>> from django import forms
+    >>> class UserForm(forms.Form):
+    ...     username = forms.CharField(max_length=10, help_text='e.g., user@example.com')
+    >>> f = UserForm()
+    >>> print(f.as_div())
+    <div><label for="id_username">Username:</label>
+    <div class="helptext" id="id_username_helptext">e.g., user@example.com</div>
+    <input type="text" name="username" maxlength="10" required aria-describedby="id_username_helptext" id="id_username">
+    </div>
+    
+When adding a custom ``aria-describedby`` attribute, make sure to also include the id of the ``help_text`` element (if used) in the correct order.
```

Comment:
All the docs should be wrapped at 79 columns. Also, the sentence above says "in the correct order", how can a user knows what the correct order is? (I for one do not know :-))

Reply:
I have changed correct -> desired and wrapped the line.